bundles/org.openhab.binding.pirateweather/README.md

```diff
@@ -0,0 +1,345 @@
+# Pirate Weather Binding
+
+This binding integrates the [Pirate Weather API](https://pirate-weather.apiable.io/).
+
+## Supported Things
+
+There are two supported things.
+
+### Pirate Weather Account
+
+First one is a bridge `weather-api` which represents the Pirate Weather account.
+The bridge holds the mandatory API key to access the Pirate Weather API and several global configuration parameters.
+If your system language is supported by the Pirate Weather API it will be used as default language for the requested data.
```

Comment:
Need to look at the actual handler, but it would be best to 1. use the openHAB configured language, 2. fallback to the system language and 3. use english of none is detectable.

Maybe it was already coded like this, but in that case the documentation should be update or it should be clear that with system you mean openHAB.